model/patch/patch.go

```diff
@@ -578,6 +578,18 @@ func (p *Patch) SetActivated(ctx context.Context, versionId string) error {
 	return err
 }
 
+// SetChildPatches appends the IDs of downstream patches to the db
+func (p *Patch) SetChildPatches(ctx context.Context) error {
+	triggersKey := bsonutil.GetDottedKeyName(TriggersKey, TriggerInfoChildPatchesKey)
+	_, err := evergreen.GetEnvironment().DB().Collection(Collection).UpdateOne(ctx,
```

Comment:
This is a long form syntax that we don't typically use; UpdateOne in this package is better (see the below function for an example)